application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/JSONService.xml

```diff
@@ -109,7 +110,15 @@
   #set($filterDoc = $!{request.calendarDoc})
 #end
 ##
-#set($events = $services.moccacalendar.queryEvents($startQuery, $endQuery, "$!{request.filter}", $filterDoc, $orderAsc))
+#if ($wikis != '')
+  #set ($wikiIds = [])
+  #foreach ($wikiId in $wikis.split(','))
+    #set ($discard = $wikiIds.add($wikiId.trim()))
+  #end
```

Comment:
Request parameters can have multiple values. Why not use that instead of passing a single comma-separated value?

Reply:
Indeed, I have turned this parameter to a list of values. I added a CSRF check in addition.